PayloadPkg/Library/SblParameterLib/SblParameterLib.c

```diff
@@ -327,7 +329,8 @@ AddSblCommandLine (
   AsciiSPrint (ParamValue, sizeof (ParamValue), " bdev=%a", GetBootDeviceNameString (BootOption->DevType));
   AsciiStrCatS (CommandLine, MaxCmdSize, ParamValue);
 
-  AsciiSPrint (ParamValue, sizeof (ParamValue), " diskbus=0x%x", BootOption->DevAddr);
+  DiskBus = GetDeviceAddr (BootOption->DevType, BootOption->DevInstance, FALSE);
```

Comment:
What is disk bus ?  I  thought the value returned is device MMIO address.

Reply:
required specific for Android OS.